### ISSUE TITLE
[BUG] Missing dependencies with GUI

### ISSUE THREAD

**Issue by camp0** [Chia-Network/chia-blockchain#1415]
**Describe the bug**
While executing the command npm run electron some dependencies packages in python are missing, one is websocket that is easiliy solve by installing it "pip install websocket" but later there is a path dependencies that is failing.

``
Running python script
Script ../src/daemon/server.py
child process success
(node:2940) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
stderr: Traceback (most recent call last):
  File "/home/luis/repositories/chia-blockchain/chia-blockchain-gui/../src/daemon/server.py", line 18, in <module>
stderr:     from src.cmds.init import chia_init
ModuleNotFoundError: No module named 'src'
closing code: 1
``

**To Reproduce**

1. Go to 'cd chia-blockchain-gui'
2. Click on 'run npm run electron.'

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS: Linux
- OS Version/Flavor: Fedora33
- CPU: [e.g. Intel Xeon 8175M]

**Additional context**

**Comment by zachlankton** [comment]
Just double checking... did you run 
```bash
sh install.sh

. ./activate

sh install-gui.sh

cd chia-blockchain-gui
npm run electron &
```

**Comment by camp0** [comment]
Yes I did that before running the command

**Comment by zachlankton** [comment]
Are you running fedora 33 server or workstation? I'm going to try recreating on my local machine.

**Comment by camp0** [comment]
Yes im on the latest version of chia and on fedora33 lxd desktop, just git pull this morning and also run the "npm run build", from my point of view it looks that some path is missing on the running of server.py, trying to fix it but with no success now, keep you up to date if I make some progress there.

**Comment by sa970** [comment]
May you send the numbers

Sent from my iPhone

‫في 21‏/03‏/2021 الساعة 6:57 م، كتب/كتبت ‏Luis ***@***.***>:‬

﻿

Yes im on the latest version of chia and on fedora33 lxd desktop, just git pull this morning and also run the "npm run build", from my point of view it looks that some path is missing on the running of server.py, trying to fix it but with no success now, keep you up to date if I make some progress there.

**Comment by camp0** [comment]
Hi All,

`(venv) [luis@localhost chia-blockchain-gui]$ cat /etc/redhat-release 
Fedora release 33 (Thirty Three)
`

I make it working by adding this line on the server.py file

`sys.path.append("/home/luis/repositories/chia-blockchain/chia-blockchain-gui/../")
sys.path.append("/home/luis/repositories/chia-blockchain/chia-blockchain-gui/./")
from src.cmds.init import chia_init
from src.daemon.windows_signal import kill
from src.server.server import ssl_context_for`

Also I instaled some packages that were missing or not installed correctly on the process of installation

`pip install click pyyaml blspy clvm clvm_rs clvm_tools cryptography bitstring keyrings.cryptfile aiohttp colorlog concurrent_log_handler`

I dont get errors on the python side but got a "start failed"
`(venv) [luis@localhost chia-blockchain-gui]$ npm run electron &
[1] 7186
(venv) [luis@localhost chia-blockchain-gui]$ 
> chia-blockchain@ electron /home/luis/repositories/chia-blockchain/chia-blockchain-gui
> electron .

/home/luis/repositories/chia-blockchain/app.asar.unpacked/daemon
false
Error loading config - using defaults
/home/luis/repositories/chia-blockchain/app.asar.unpacked/daemon
false
/home/luis/repositories/chia-blockchain/app.asar.unpacked/daemon
false
Running python script
Script ../src/daemon/server.py
child process success
(node:7208) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
No cert
Have cert
`

Thanks for the fast response :)
